index.js

```diff
@@ -142,7 +219,38 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
```

Comment:
I think we did a lot more work in this one than we needed to. Instead of using three for loops, I think we could have just looped once and created a variable that would hold the largest box office and corresponding movie object respectively, and then continued to conditionally check and update these values.